src/main/java/com/ebay/myriad/configuration/MyriadConfiguration.java

```diff
@@ -169,6 +179,19 @@ public String getNativeLibrary() {
         return Strings.isNullOrEmpty(this.nativeLibrary) ? DEFAULT_NATIVE_LIBRARY : this.nativeLibrary;
     }
 
+    public String getResourceManagerHost() {
```

Comment:
If the Myriad scheduler runs as a plugin to the RM, is there any way that they could be different hosts/IPs?
Or are you just trying to specify an alternate hostname/IP to resolve instead of localhost/127.0.0.1?